src/omg-cli/omg/commands/commands.go

```diff
@@ -90,14 +94,15 @@ func retry(fn step, times int) error {
 	errs := []error{}
```

Comment:
I'm not sure if this function is used. Looking for it in GitHub only brings up the one instance.

Reply:
Deleted